packages/@okta/i18n/src/properties/login_ok_PL.properties

```diff
@@ -612,7 +615,7 @@ oie.password.challenge.title = 》Ṽéŕîƒý ŵîţĥ ýöûŕ þåššŵöŕ
 oie.password.enroll.title = 》Šéţ ûþ þåššŵöŕð ฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.password.passwordLabel = 》Éñţéŕ þåššŵöŕð โ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.password.confirmPasswordLabel = 》Ŕé‐éñţéŕ þåššŵöŕð 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
-oie.password.reset.verification = 》Ṽéŕîƒý ŵîţĥ öñé öƒ ţĥé ƒöļļöŵîñĝ åûţĥéñţîçåţöŕš ţö ŕéšéţ ýöûŕ þåššŵöŕð· 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+oie.password.reset.verification = 》Ṽéŕîƒý ŵîţĥ öñé öƒ ţĥé ƒöļļöŵîñĝ šéçûŕîţý ɱéţĥöðš ţö ŕéšéţ ýöûŕ þåššŵöŕð· 홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
```

Comment:
These were altered as part of #2080, but the dev (me) didn't run the pseudo-loc task. These are **safe** and **valid** changes.